web-ui/src/app/components/adress/adress.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-adress',
```

Comment:
Maybe instead of `app` use some other prefix like `gretel` (hmm long) or just `g`?